### DISCUSSION TITLE
PAG支持倒放吗

### DISCUSSION THREAD

**Discussion by 211sx** [Tencent/libpag#1561] [Q&A] [answered]
如果我真想用一个PAG格式文件实现倒放，用setProgress、flush方法，该如何实现。

**Comment by FusionXu** [comment, marked as answer]
你可以使用PAGPlayer来控制文件播放
PAGPlayer每一帧的绘制，都需要setProgress，只要设置的进度改成从1.0到0.0，这样就是倒叙播放了
但要注意的是，如果包含bmp预合成的PAG素材，则不建议使用倒叙播放，倒叙解码视频在没有做特殊兼容的情况下，性能表现是很差的
这时候不如直接做一个倒叙的PAG素材
